lib/offline.js

```diff
@@ -63,52 +70,111 @@ module.exports = function(config) {
         return '';
       }
     }).then(function(cacheId) {
-      var staticFileGlobs = fileGlobs.map(function(v) {
+      checkImports(importScripts, rootDir);
+
+      var absoluteGlobs = fileGlobs.map(function (v) {
         return path.join(rootDir, v);
       });
+      var files = flatGlobs(absoluteGlobs);
+      var filesAndHashes = getFilesAndHashes(files, rootDir, absoluteGlobs);
 
-      staticFileGlobs.forEach(function(globPattern) {
-        glob.sync(globPattern.replace(path.sep, '/')).forEach(function(file) {
-          var stat = fs.statSync(file);
+      var replacements = {
+        cacheId: cacheId,
+        ignoreUrlParametersMatching: config.ignoreUrlParametersMatching || [/./], //TODO: Not implemented in template
```

Comment:
Rather than labeling this line TODO, I'd remove it and file a followup issue to reimplement it in the new template. It isn't important at the moment, since GitHub Pages only supports static content. Although it might be useful in the future for offlining an app and then hosting it with another hosting provider (per #24).